packages/@o3r/core/schematics/rule-factories/o3r-basic-updates/index.ts

```diff
@@ -11,37 +12,50 @@ import { ngAddPackages, readAngularJson, readPackageJson } from '@o3r/schematics
  */
 export function o3rBasicUpdates(pName: string | null | undefined, o3rCoreVersion?: string, projectType?: 'application' | 'library'): Rule {
 
+  const ngUpdateScript = (tree: Tree, packageJsonObj: PackageJson, packageJsonPath = '/package.json') => {
+    if (!packageJsonObj.scripts) {
+      packageJsonObj.scripts = {};
+    }
+
+    packageJsonObj.scripts['update:otter'] = 'ng update @o3r/core';
+
+    tree.overwrite(packageJsonPath, JSON.stringify(packageJsonObj, null, 2));
+  };
+
   const updatePackageJson = (tree: Tree, _context: SchematicContext) => {
     const workspace = readAngularJson(tree);
 
+    if (!projectType) { // at the root of the monorepo
+      const rootPackageJsonPath = '/package.json';
+      const rootPackageJsonString = tree.read(rootPackageJsonPath)?.toString();
+      if (rootPackageJsonString) {
+        const rootPackageJsonObject = JSON.parse(rootPackageJsonString) as PackageJson;
+        ngUpdateScript(tree, rootPackageJsonObject, rootPackageJsonPath);
+      }
+    }
+
     Object.entries(workspace.projects)
       .filter(([name]) => !pName || pName === name)
       .forEach(([,workspaceProject]) => {
         const packageJson = readPackageJson(tree, workspaceProject);
-        if (!packageJson.scripts) {
-          packageJson.scripts = {};
-        }
-
-        packageJson.scripts['update:otter'] = 'ng update @o3r/core';
-
-        tree.overwrite(`${workspaceProject.root}/package.json`, JSON.stringify(packageJson, null, 2));
+        ngUpdateScript(tree, packageJson, `${workspaceProject.root}/package.json`);
       });
     return tree;
   };
 
   return async (tree: Tree, context: SchematicContext) => {
     // eslint-disable-next-line @typescript-eslint/naming-convention
     const { NodeDependencyType } = await import('@schematics/angular/utility/dependencies');
-
+    const packagesToAddByProjectType = projectType ? ['@ama-sdk/core'] : ['@ama-sdk/schematics'];
```

Comment:
Not sure you need to add `@ama-sdk/core` in case of package

Reply:
we use it for utils.datetime so I think we can add it